ci/integration.yml

```diff
@@ -0,0 +1,23 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.8-alpine
+inputs:
+  - name: pr
+  - name: secrets
+run:
+  path: sh
+  args:
+    - -e
+    - -c
+    - |
+      export ACCESS_TOKEN=$(awk -F':' '$1 ~ /compose_access_token/ {print $2}' ./secrets/compose-broker-secrets.yml)
+      mkdir -p "${GOPATH}/src/github.com/alphagov/paas-compose-broker"
+      cp -r  pr/* "${GOPATH}/src/github.com/alphagov/paas-compose-broker/"
+      cd "${GOPATH}/src/github.com/alphagov/paas-compose-broker"
+      apk update && apk add git
+      go get github.com/onsi/ginkgo/ginkgo
```

Comment:
Even better would be to avoid using the ginkgo CLI at all. If we can run this using `go test` we avoid these versioning issues as it will exclusively use the vendored ginkgo libraries.